app/controllers/concerns/attachment_manageable.rb

```diff
@@ -1,6 +1,6 @@
 module AttachmentManageable
-  def allow_only_one_attachment
-    return unless params[:files].is_a?(Array)
+  def allow_only_one_csv_attachment
```

Comment:
This method could be made more expandable by declaring a parameter to define the file type when used, but given we don't need that anywhere, this is fine for now, I think.